v1/6_security.md

```diff
@@ -1,13 +1,49 @@
 # Security
 
-## Denial of Service (DoS) attacks
+In order to keep the protocol secure, we need to mitigate the risks associated with malicious actors and the risk of losing funds.
+This section outlines the security measures in place to protect the integrity of the Coinswap protocol, the possible abort scenarios and the mechanisms to prevent them or penalize the malicious actors.
 
-A denial of service (DoS) attack is a type of attack where a malicious actor floods the network with a large number of requests, causing the network to become overwhelmed and unresponsive. In the context of Coinswap, a DoS attack could disrupt the swap process and prevent legitimate users from participating in the network.
+## Aborts
 
-## Distributed Denial of Service (DDoS) attacks
+An abort is a scenario where one of the participants in the swap process fails to complete their part of the transaction or disconnects from the network.
 
-A distributed denial of service (DDoS) attack is a type of attack where multiple compromised systems are used to flood the network with requests, causing a denial of service. DDoS attacks are more difficult to mitigate than DoS attacks because they involve multiple sources of traffic. In the context of Coinswap, a DDoS attack could disrupt the swap process and prevent legitimate users from participating in the network.
+### 1. Taker aborts after full setup
+
+This is the situation where the Taker drops connection after broadcasting all the funding transactions.
+The Makers identifies this and waits for a timeout for the Taker to come back.
+If the Taker doesn't come back within timeout, the Makers broadcasts the contract transactions and reclaims their funds via timelock.
+The Taker after coming live again will see unfinished coinswaps in his wallet. He can reclaim his funds via broadcasting his contract transactions and claiming via timelock.
+
+### 2. Maker Drops Before Setup
```

Comment:
I guess , it would be better to mention all possible cases under this scenario , as similarly done for `Maker drop After Setup`.